Description: keeps quality scores.
 This is a transient patch for my personal use, but that I would like to share with others.
 I will discuss with Upstream about the possibiltiy to have it in the next official patch release.
Origin: http://lists.open-bio.org/pipermail/emboss/2009-September/003735.html
--- emboss-6.1.0.orig/ajax/ajseq.c
+++ emboss-6.1.0/ajax/ajseq.c
@@ -782,6 +782,12 @@
     if (seq->Fttable)
 	pthis->Fttable = ajFeattableCopy(seq->Fttable);
 
+    if (seq->Accuracy)
+    {
+        AJCNEW0(pthis->Accuracy,seq->Seq->Len);
+        memmove(pthis->Accuracy,seq->Accuracy,seq->Seq->Len*sizeof(float));
+    }
+
     return pthis;
 }
 
